x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -265,6 +262,16 @@ export class ESGeoGridSource extends AbstractESAggSource {
     return convertRegularRespToGeoJson(esResponse, this._descriptor.requestType);
   }
 
+  _buildGeotileGridDsl(precision, bounds) {
+    return makeGeotileGridDsl(this._descriptor.geoField, bounds, precision);
```

Comment:
Should bounds be included for geo_point type? I thought that had a performance impact and is not needed? https://github.com/elastic/kibana/issues/56876#issuecomment-595317196

Reply:
I discussed this offline with @talevy . we agreed that for simplicity we can keep it for `geo_point` as well.